.github/workflows/CI.yml

```diff
@@ -1,6 +1,6 @@
 name: Rust CI
```

Comment:
Can you just maintain the name `CI` instead of `Rust CI`. You don't need to really specify the `Rust` in the name for the workflow to work.